packages/webdriver/src/bidi/core.ts

```diff
@@ -27,7 +27,7 @@ export class BidiCore {
     constructor (webSocketUrl: string, opts?: ClientOptions) {
         this.#webSocketUrl = webSocketUrl
         log.info(`Connect to webSocketUrl ${this.#webSocketUrl}`)
-        this.#ws = new environment.value.Socket(this.#webSocketUrl, opts) as unknown as WebSocket
+        this.#ws = new environment.value.Socket(this.#webSocketUrl, { ...opts }) as unknown as WebSocket
```

Comment:
Why do we spread this when nothing new is added (spreading it like this only costs extra memory)?

Reply:
Thanks for pointing that out, didn't knew that! Kind of new to TS.

I will remove it and the other occurrence as well.